### PR TITLE
minor: fix violations for RedundantThisCheck

### PR DESCRIPTION
**Pull request by wilcoln**
As discussed in https://github.com/checkstyle/checkstyle/pull/8013#issuecomment-610248687

### CODE REVIEW

File: eclipsecs-sevntu-plugin-feature/.project [hunk 1]
```diff
@@ -10,8 +10,14 @@
 			<arguments>
 			</arguments>
 		</buildCommand>
+		<buildCommand>
+			<name>org.eclipse.m2e.core.maven2Builder</name>
```
Comment (rnveach) [inline eclipsecs-sevntu-plugin-feature/.project:14]: This isn't an expected change.

Comment (wilcoln) [reply eclipsecs-sevntu-plugin-feature/.project]: Done.

File: eclipsecs-sevntu-plugin-feature/.project [hunk 2]
```diff
@@ -14,4 +14,4 @@
 	<natures>
 		<nature>org.eclipse.pde.FeatureNature</nature>
 	</natures>
-</projectDescription>
+</projectDescription>
```
Comment (rnveach) [inline eclipsecs-sevntu-plugin-feature/.project:17]: Same.

Comment (wilcoln) [reply eclipsecs-sevntu-plugin-feature/.project]: Done.

**Comment by romani** [comment]
this project is following main repo style, so we need this update.

**Pull request merged**